app/Livewire/GlobalSearch.php

```
@@ -0,0 +1,371 @@
+                    ];
+                });
+
+            // Merge all collections
+            $items = $items->merge($applications)
+                ->merge($services)
+                ->merge($databases)
+                ->merge($servers);
+
+            return $items->toArray();
+        });
+    }
```

PHPMD (2.15.0)

The method loadSearchableItems() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)